Modules/OptionHolder.cs

```diff
@@ -2909,6 +2913,17 @@ public static void Load()
         SetupLoversRoleOptionsToggle(23600);
         
         SetupAdtRoleOptions(23700, CustomRoles.Reach, canSetNum: true);
+
+        SetupAdtRoleOptions(27300, CustomRoles.Rainbow, canSetNum: true, tab: TabGroup.Addons);
+        //RainbowColorChangeTime = FloatOptionItem.Create(27303, "RainbowColorChangeTime", new(0.1f, 5f, 2.5f), 0.5f, TabGroup.Addons, false)
+        //    .SetParent(CustomRoleSpawnChances[CustomRoles.Rainbow])
+        //    .SetValueFormat(OptionFormat.Seconds);
+        CrewCanBeRainbow = BooleanOptionItem.Create(27303, "CrewCanBeRainbow", true, TabGroup.Addons, false)
+            .SetParent(CustomRoleSpawnChances[CustomRoles.Rainbow]);
+        ImpCanBeRainbow = BooleanOptionItem.Create(27304, "ImpCanBeRainbow", true, TabGroup.Addons, false)
+            .SetParent(CustomRoleSpawnChances[CustomRoles.Rainbow]);
+        NeutralCanBeRainbow = BooleanOptionItem.Create(27305, "NeutralCanBeRainbow", true, TabGroup.Addons, false)
+            .SetParent(CustomRoleSpawnChances[CustomRoles.Rainbow]);
```

Comment:
Please, when adding a role or add-on, create separate files for them, otherwise the code for a specific role will have to be searched throughout the code